test/melange-testing-library/README.md

```diff
@@ -0,0 +1,5 @@
+### vendored melange-testing-library
```

Comment:
I read the PR description but it explains what's being done but not why.

What was the issue with using react test renderer? It's a much lighter solution than react testing library and it should catch any runtime issues just fine.

Reply:
ftr: https://react.dev/warnings/react-dom-test-utils#rest-of-reactdomtestutils-apis